### PR TITLE
LOG-1466: Modify CI jobs of ElasticSearch Operator and ElasticSearch Proxy to consume log exporter image

### PR DESCRIPTION
**Pull request by vparfonov**
Modify CI jobs to consume log exporter image: 

- ElasticSearch Operator 
- ElasticSearch Proxy

Ref: https://issues.redhat.com/browse/LOG-1466 

Signed-off-by: Vitalii Parfonov <vparfono@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-master.yaml [hunk 1]
```diff
@@ -125,6 +129,8 @@ tests:
         name: pipeline:logging-elasticsearch6
       - env: IMAGE_LOGGING_KIBANA6
         name: pipeline:logging-kibana6
+      - env: IMAGE_LOG_FILE_METRIC_EXPORTER
```
Comment (jcantrill) [inline ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-master.yaml:132]: Not needed here as this is the e2e test for the EO that does not invoke the CLO logic

File: ci-operator/config/openshift/elasticsearch-proxy/openshift-elasticsearch-proxy-master.yaml [hunk 2]
```diff
@@ -156,6 +162,8 @@ tests:
         name: pipeline:logging-elasticsearch6
       - env: IMAGE_LOGGING_KIBANA6
         name: pipeline:logging-kibana6
+      - env: IMAGE_LOG_FILE_METRIC_EXPORTER
```
Comment (jcantrill) [inline ci-operator/config/openshift/elasticsearch-proxy/openshift-elasticsearch-proxy-master.yaml:165]: not required here as it is not needed for EO testing

File: ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-master.yaml (hunk 1 above)
Comment (vparfonov) [reply ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-master.yaml:132]: fixed

File: ci-operator/config/openshift/elasticsearch-proxy/openshift-elasticsearch-proxy-master.yaml (hunk 2 above)
Comment (vparfonov) [reply ci-operator/config/openshift/elasticsearch-proxy/openshift-elasticsearch-proxy-master.yaml:165]: fixed, thx

File: ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-master.yaml (hunk 1 above)
Comment (vparfonov) [reply ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-master.yaml]: fixed, thx

File: ci-operator/config/openshift/elasticsearch-proxy/openshift-elasticsearch-proxy-master.yaml (hunk 2 above)
Comment (vparfonov) [reply ci-operator/config/openshift/elasticsearch-proxy/openshift-elasticsearch-proxy-master.yaml]: fixed

**Pull request merged**